### PR TITLE
WIP: try to repro/resolve #1094

### LINKED ISSUE #1094: revokeInvitatiion not working
I tried to delete an existing invitation but I get an error

mutation:
```graphql
mutation revokeInvitation($id: UserInvitationId!) {
          revokeUserInvitation(input: { id: $id }) {
            invitation {
              id
            }
          }
        }
```

with variables
```json
{
    "id": "10d"
}
```

And produces an error:
```json
[
    {
        "message": "Invitation does not exist, is no longer pending, or was issued by someone else.",
        "extensions": {
            "odb_error": {
                "odb_error.tag": "invitation_error",
                "odb_error.detail": "Invitation does not exist, is no longer pending, or was issued by someone else.",
                "odb_error.data": {
                    "invitationId": "10d"
                }
            }
        }
    }
]
```

### PR DESCRIPTION
**Pull request by tpolecat**
WIP: trying to repro/resolve #1094

### CODE REVIEW

**Comment by tpolecat** [comment]
@cquiroz can you compare this test with what you're doing? I can't get it to fail.

File: modules/service/src/test/scala/lucuma/odb/graphql/mutation/revokeUserInvitation.scala [hunk 1]
```diff
@@ -78,6 +78,56 @@ class revokeUserInvitation extends OdbSuite {
...
+          .hcursor
+          .downField("program")
+          .downField("userInvitations")
+          .downArray
+          .downField("id")
+          .require[UserInvitation.Id]
+
+        expect(
+          user = pi,
+          query = revoke(id),
```
Comment (cquiroz) [inline modules/service/src/test/scala/lucuma/odb/graphql/mutation/revokeUserInvitation.scala:107]: What does `revoke` exactly do?

Comment (tpolecat) [reply modules/service/src/test/scala/lucuma/odb/graphql/mutation/revokeUserInvitation.scala:107]: https://github.com/gemini-hlsw/lucuma-odb/blob/0b7f5a882f79b2b811e729d070c0bccac4dc4a7d/modules/service/src/test/scala/lucuma/odb/graphql/mutation/revokeUserInvitation.scala#L27

**Comment by cquiroz** [comment]
Ok, I keep getting the same error
I have created invitations with a certain id which I get with a regular program query. But when I try to delete using the keyy it says invitation does not exist
Not sure if it matters but to create the invitation I only pass the program id and email

**Comment by tpolecat** [comment]
Hm. Can you show me the mutation you use to create the invitation?

**Comment by tpolecat** [comment]
Might as well keep the test.

**Review by swalker2m — APPROVED** [review]

**Pull request merged**